src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Movies = {
+  movies: Movie[],
+};
+
+export const MoviesList: React.FC<Movies> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard {...movie} />
```

Comment:
Don't forget about `key` attribute when rendering list.

Reply:
Aren't we using keys for the list components? I thought we should add them to the MovieCard

Reply:
Keys are added in the same component, where `map` is used, to the top level tag of map return.